tests/Unit/Http/Action/AbstractActionTestCase.php

```diff
@@ -20,11 +20,16 @@
 use Symfony\Component\Routing;
 use Twig_Environment;
 
+/**
+ * @deprecated
+ */
 abstract class AbstractActionTestCase extends Framework\TestCase
```

Comment:
Since all this abstract test case does is

* import the `Helper` trait
* provide methods which create mocks

we should probably remove it eventually.